README.md

```diff
@@ -29,6 +29,8 @@ Custom React PropType validators that we use at Airbnb. Use of [airbnb-js-shims]
  - `componentWithName`: restrict the prop to only allow a component with a certain name/displayName. Accepts a string, or a regular expression. Also accepts an `options` object with an optional `stripHOCs` array of string HOC names to strip off before validating; an HOC name must not contain parentheses and must be in camelCase.
    - `foo: componentWithName('Component')`
    - `foo: componentWithName('Component', { stripHOCs: ['withDirection', 'withStyles'] })`
+ - `disallowedIfOtherPropType`: restrict the prop from being provided if the given other prop name matches the given other prop type.
```

Comment:
i think `disallowedIf` is a good enough name, since all of the existing "if" or "by" validators take a type